packages/phone-number-privacy/signer/package.json

```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "@celo/contractkit": "1.0.0-beta3",
```

Comment:
do the `@celo/` dependencies need to be updated?

Reply:
In this case, the package released for the phone-number-privacy-commons uses the `1.0.0-beta3`, and as we require to use that fix version for the other phone-...-packages, there was a conflict of types between the 1.0.0-beta3 in the -common, and our 1.0.2.
The next time we release the `-common` we could fix that, but in this particular case, it was not required for the PR to work. Is something that we could add for the next time we require to release all the phone packages